imagemanager/imagemanager.go

```diff
@@ -838,6 +845,10 @@ func (imagemanager *Imagemanager) removeOutdatedService(serviceID string) error
 		}
 	}
 
+	if err := imagemanager.state.Remove(serviceID); err != nil {
```

Comment:
if we can't remove service storage, we have to try remove state

Reply:
Done